cms/static/sass/elements/_modules.scss

```diff
@@ -168,18 +168,49 @@
 
     // specific menu types
     &.new-component-problem {
-      padding-bottom: ($baseline/2);
```

Comment:
Note to reviewers: did some adjustments so that the new component menu renders with the same spacing around the "Cancel" button and support legend for Advanced Modules, HTML, and Problems. Previously there were some vertical spacing differences.